### PR TITLE
Improved README, added Travis CI badge

### PR DESCRIPTION
**Pull request by LukasRychtecky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-see [docs](http://django-pyston.readthedocs.org/)
+# django-pyston
+
+[![Build
+Status](https://travis-ci.org/druids/django-pyston.svg?branch=master)](https://travis-ci.org/druids/django-pyston)
+
+Django-pyston is library originally created from `django-piston` application
```
Comment (asgeirrr) [inline README.md]: is **a** library

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+(https://github.com/django-piston/django-piston), but currently the code was fully rewritten. It’s purpose is help with
```
Comment (asgeirrr) [inline README.md]: code **is** fully rewritten. It’s purpose is **to** help with

File: README.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+developing RESTful applications.
```
Comment (asgeirrr) [inline README.md]: ...development of RESTful applications.

Comment (LukasRychtecky) [inline README.md]: Já jsem to zkopíroval z těch docs a ani si to nepřečetl :/

**Pull request merged**